src/ViewModels/WebMapServiceItemViewModel.js

```diff
@@ -108,8 +108,8 @@ var WebMapServiceItemViewModel = function(application) {
     delete this.__knockoutObservables.dataUrlType;
     knockout.defineProperty(this, 'dataUrlType', {
         get : function() {
-            if (defined(this._dataUrlType)) {
-                return this._dataUrlType;
+            if (defined(this._dataUrl)) {
+                return 'direct';
```

Comment:
Because dataUrls are now direct and wms.  The one outlier we had was removed and it now points to the ckan page rather some broken url in the json file.